tests/test_quick_run_config_generator.py

```diff
@@ -911,6 +930,138 @@ def _get_next_run_config_ensemble(self,
 
         self.assertEqual(perf_config['batch-size'], 1)
 
+    def _get_next_run_config_bls(self, max_concurrency=0, min_concurrency=0):
+        """
+        Test that get_next_run_config() creates a proper RunConfig for BLS
+
+        Sets up a case where the coordinate is [1,2,3,4,5,6], which corresponds to
```

Comment:
Is this comment correct? Seems like way more than 6 values below

Reply:
Reordered